charts/kong/values.yaml

```diff
@@ -130,6 +130,20 @@ proxy:
     parameters:
     - http2
 
+  # Define stream (TCP) listen
+  # To enable, remove "{}", uncomment the section below, and select your desired
+  # ports and parameters.
+  stream: {}
+    # - name: example-stream
+    #   containerPort: 9000
+    #   servicePort: 9000
+    #   # Optionally set a static nodePort if the service type is NodePort
+    #   # nodePort: 32080
+    #   # Additional listen parameters, e.g. "ssl", "reuseport", "backlog=16384"
+    #   # "ssl" is not supported on versions <2.0, but is not required for
+    #   # SNI-based listens. It is used to reject any non-TLS connections.
```

Comment:
I misread http://nginx.org/en/docs/stream/ngx_stream_core_module.html#listen re requiring TLS. Not sure about our support for it. The changelog suggests that it was only added as of 2.0, whereas https://docs.konghq.com/1.3.x/admin-api/#add-route indicates it is supported. Changelog is probably more trustworthy, so assuming 2.0+ is the only option unless I hear otherwise from the core team.